src/main/java/de/adesso/kicker/notification/NotificationController.java

```diff
@@ -0,0 +1,38 @@
+package de.adesso.kicker.notification;
+
+import de.adesso.kicker.user.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
```

Comment:
You can make this a normal `@Controller` as the frontend wants to use Thymeleaf and you can change the `@DeleteMappings` to `@GetMapping` or `@PostMapping` depending on what you want to do there.